### ISSUE TITLE
[WARNING] There are local changes not commited to Git

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#651]
In GitLab by @mcastril on Jan 28, 2021, 17:31

Hi @dbeltrankyl 

@eferre1 reported me that copying experiment `t0gg` is showing up this warning, that should not appear since this experiment is a production/release one.

`autosubmit expid --copy t0gg --HPC marenostrum4 --description "Autosubmit basics tutorial running EC-Earth"`

```
Configuration files OK
checking model version...
test
[WARNING] There are local changes not commited to Git
Experiment registered successfully
Remember to MODIFY the config files!
```

I went to verify it:

```
mcastril@ bscearth396:/esarchive/autosubmit/t0gg/proj/auto-ecearth3> git status
On branch trunk
Your branch is up-to-date with 'origin/trunk'.
nothing to commit, working tree clean
```

I also checked how Autosubmit is checking:

`mcastril@ bscearth396:/esarchive/autosubmit/t0gg/proj/auto-ecearth3> git diff-index HEAD --`

Which outputs nothing.

Something that we are not seeing?

```
# Autosubmit version identifier
AUTOSUBMIT_VERSION = 3.12.0

```

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jan 29, 2021, 11:37

Hi @mcastril , 

it happens also on 3.13.0?

**Comment by kinow** [comment]
In GitLab by @mcastril on Jan 29, 2021, 14:03

We didn't try @dbeltrankyl . As it is a 3.3.3 experiment from the testing suite users should be using 3.12.0 because it's a production one.

**Comment by kinow** [comment]
In GitLab by @mcastril on Feb 10, 2021, 15:31

We consider it as fixed in 3.13.0

**Comment by kinow** [comment]
In GitLab by @mcastril on Feb 10, 2021, 15:31

closed
